RadixWallet/Features/SettingsFeature/DebugSettings/Coordinator/DebugSettingsCoordinator+View.swift

```diff
@@ -1,8 +1,16 @@
 import ComposableArchitecture
 import SwiftUI
 
+extension DebugSettingsCoordinator.State {
+	var viewState: DebugSettingsCoordinator.ViewState {
+		.init()
+	}
+}
+
 // MARK: - DebugSettingsCoordinator.View
 extension DebugSettingsCoordinator {
+	public struct ViewState: Equatable {}
```

Comment:
Redundant? why was this added?

Reply:
The reason I added this empty `ViewState` is because the new `SettingsRow` now requires a `ViewStore` in order to send the corresponding action when the rows are tapped. Using an empty `ViewState` is the easiest way I've found to pass a `ViewStore` (and not the `Store` we already have), but open to suggestions if you have a better way of doing so.

Reply:
Well, actually we could always send `Store` and just do `store.send(.view(action))`. Will do that

Reply:
yup, you can just use store :)

Reply:
Yeah, I am doing that for all new features, and I have stopped making ViewState in many cases, since it is going away.